plenum/server/view_change/view_changer.py

```diff
@@ -225,6 +226,14 @@ def view_change_in_progress(self) -> bool:
     def view_change_in_progress(self, value: bool):
         self._view_change_in_progress = value
 
+    @property
+    def pre_view_change_in_progress(self) -> bool:
+        return self._pre_view_change_in_progress
+
+    @pre_view_change_in_progress.setter
+    def pre_view_change_in_progress(self, value: bool):
+        self._pre_view_change_in_progress = value
+
```

Comment:
Do we really need to make this a property if we are just using trivial getter and setter without any additional logic?

Reply:
As a quick fix, I did it similar to `view_change_in_progress`, but yes, agree.

Reply:
Done